docs/python/how-to-guides/use-uris.md

```diff
@@ -28,15 +45,42 @@ The above URL can be broken down as follows:
 - Path
   - The path, in this case, is `/core/docs`. It is a path on the authority.
 
+### Deephaven URI structure
+
 Deephaven URIs use a similar syntax:
 
-`dh+plain://<authority>/<path>`
+`dh+plain://<authority>[:<port>]/<scope>/<table_name>`
+
+The components are:
+
+- **`dh+plain`** is the scheme.
+  - This is **always** `dh+plain` for all Deephaven table URIs.
+  - The scheme identifies the protocol for accessing Deephaven tables.
+  - This has no relation to application types (script, static, dynamic, qst) which are configuration options in [Application Mode](./application-mode.md).
```

Comment:
I think I wrote this a while back, but I don't really know what this last bullet point means.